utils/spdx-document/src/main/kotlin/model/SpdxPackage.kt

```diff
@@ -125,13 +125,13 @@ data class SpdxPackage(
      * The concluded license for the package as SPDX expression. To represent a not present value [SpdxConstants.NONE]
      * or [SpdxConstants.NOASSERTION] must be used.
      */
-    val licenseConcluded: String,
+    val licenseConcluded: String = SpdxConstants.NOASSERTION,
 
     /**
      * The declared license for the package as SPDX expression. To represent a not present value [SpdxConstants.NONE] or
      * [SpdxConstants.NOASSERTION] must be used.
      */
-    val licenseDeclared: String,
+    val licenseDeclared: String = SpdxConstants.NOASSERTION,
```

Comment:
Slightly related: I believe that `SpdxPackage.toPackage()` needs two fixes.

1. Concluded license should be nullified only in case of `NOASSERTION` but be kept in case of `NONE`
2.  `declaredLicenses = setOf(licenseDeclared)` should be changed to filter out `NOASSERTION`

Would you agree?